Wordle.Api/Wordle.Api/Services/PlayerService.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Wordle.Api.Data;
+
+namespace Wordle.Api.Services
+{
+    public class PlayerService
+    {
+        private readonly AppDbContext _db;
+        public PlayerService(AppDbContext db)
+        {
+            _db = db;
+        }
+
+        public async Task<IEnumerable<Player>> GetTopTen()
```

Comment:
Alot of the changes from this file were from Assignment 3. Make sure to only include changes required for Assignment 4.